docker_entrypoint/osmaxx/conversion_service/entrypoint.sh

```diff
@@ -4,7 +4,7 @@ set -e
 # wait for at most 30s for the db to be up
 ${HOME}/entrypoint/wait-for-it.sh ${DATABASE_HOST}:${DATABASE_PORT} -t 30
 
-python3 osmaxx_conversion_service/manage.py migrate && \
+python3 osmaxx_conversion_service/manage.py migrate --no-input && \
```

Comment:
> there is no need to do it my way :-)

What hasn't to be done what way? Is this about `manage.py migrate --no-input` or was that a comment for a different line?